File: problem80.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        # run all the way 
        length = 1
        k = 0
        index = 0
        for i in range(len(nums)-1):
            if length < 3:
                nums[index] = nums[i]
                index += 1
            if nums[i] == nums[i+1]:
                length += 1
            else:
                if length >= 2:
                    k += 2
                else:
                    k += 1
                length = 1
        if length == 1:
            k += 1
        else:
            k += 2
        nums[k-1] = nums[len(nums)-1]
        return k

File: problem80.cpp
#include <vector>
#include <iostream>
using namespace std;

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int k = 0;
        for(int num : nums){
            // Check k is less than two
            // if greater, check current num is not same as number two index before it
            if(k < 2 || num != nums[k-2]){
                nums[k] = num; 
                k++;
            }
        }
        return k;
    }
};